Example/README.md

```diff
@@ -1,17 +1,89 @@
-# Square Mobile Payments SDK Example App
```

Comment:
We may end up having multiple example apps so this README may need to be more generic but for now I think it's fine to have it specific to the donut counter app